packages/terra-clinical-data-grid/tests/wdio/data-grid-spec.js

```diff
@@ -0,0 +1,134 @@
+
+['medium', 'large', 'huge', 'enormous'].forEach((viewport) => {
+  describe(`DataGrid - ${viewport}`, () => {
+    before(() => browser.setViewportSize(Terra.viewports(viewport)[0]));
+
+    describe('with pinned columns and overflow columns', () => {
+      describe('with initial rendering', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-clinical-data-grid/clinical-data-grid/standard-data-grid');
+        });
+
+        Terra.should.matchScreenshot('#pinned-and-overflow', { selector: '#standard-data-grid' });
+        Terra.should.beAccessible();
+      });
+
+      describe('with horizontal overflow', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-clinical-data-grid/clinical-data-grid/standard-data-grid');
+          browser.click('[data-cell-label="section_0-0-Column-12"]');
+        });
+
+        Terra.should.matchScreenshot('#pinned-and-overflow', { selector: '#standard-data-grid' });
+      });
+
+      describe('with vertical overflow', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-clinical-data-grid/clinical-data-grid/standard-data-grid');
+          browser.click('[data-cell-label="section_0-29-Column-12"]');
+        });
+
+        Terra.should.matchScreenshot('#pinned-and-overflow', { selector: '#standard-data-grid' });
+      });
+    });
+
+    describe('with no pinned columns', () => {
+      describe('with initial rendering', () => {
+        beforeEach(() => {
+          browser.url('/#/raw/tests/terra-clinical-data-grid/clinical-data-grid/no-pinned-column-data-grid');
```

Comment:
Did we have a pinned-column example ?

Reply:
There is a test set above that includes pinned columns. There isn't a pinned-column only test, though, which might be nice to have